torch/_inductor/triton_bundler.py

```diff
@@ -192,7 +192,7 @@ def read_and_emit(
         if not TritonBundler.is_enabled():
             return None
 
-        with dynamo_timed(key="TritonBundler.read_and_emit", fwd_only=False):
+        with dynamo_timed("TritonBundler.read_and_emit", log_pt2_compile_event=False):
```

Comment:
How were you deciding whether or not to `log_pt2_compile_event` here? Because most of these ARE being logged to the event table (https://fburl.com/scuba/pt2_compile_events/f8966fla) and you are now excluding them with these changes.

Reply:
Hmm, but now I see there are some places which had `log_pt2_compile_event=True` so now I am confused if we are logging to the table or not. @jamesjwu can you weigh in? @masnesral I think I am satisfied as long as you demonstrate that the behavior doesn't change. But it does seem like we "accidentally" changed the behavior recently...

Reply:
It kind of feels like you flipped the default here, and you could avoided all the faffing about by... not flipping the default.

Reply:
Yeah because I changed the default of `log_pt2_compile_event` from False to True (because that's what we had in your doc https://docs.google.com/document/d/18ZJADPvGaKU47l7UN_DvN-7wfGyFEWIZlSs9zedaUfk). So now i pass False where it wasn't passed before and just take the default where it was pass as True before. So the intent was nothing changes as far as what's logged.